composer.json

```diff
@@ -53,7 +53,7 @@
         "aws/aws-sdk-php": "^2.7",
         "jackalope/jackalope-doctrine-dbal": "^1.1",
         "symfony/phpunit-bridge": "^2.7 || ^3.0",
-        "friendsofsymfony/rest-bundle": "^1.7.4 || ^2.0",
+        "friendsofsymfony/rest-bundle": "^2.0",
```

Comment:
We can drop a version support like this. We have to support both.

Actually this is BC break.